pkg/virt-handler/vm.go

```diff
@@ -2615,6 +2669,32 @@ func (d *VirtualMachineController) hotplugSriovInterfacesCommand(vmi *v1.Virtual
 	return nil
 }
 
+func memoryDumpPath(volumeStatus v1.VolumeStatus) string {
+	target := hotplugdisk.GetVolumeMountDir(volumeStatus.Name)
+	dumpPath := filepath.Join(target, volumeStatus.MemoryDumpVolume.TargetFileName)
+	return dumpPath
+}
+
+func (d *VirtualMachineController) getMemoryDump(vmi *v1.VirtualMachineInstance, volumeStatus v1.VolumeStatus) (bool, error) {
+	const errMsgPrefix = "failed to getting memory dump"
+
+	completed := false
+	if volumeStatus.MemoryDumpVolume != nil {
+		client, err := d.getVerifiedLauncherClient(vmi)
```

Comment:
this is in the path of updateVMIStatus. if a virt-launcher has gone down, we will not be able to update the vmi.Status field due to how this error is unwound